libtrellis/CMakeLists.txt

```diff
@@ -108,24 +108,28 @@ endif()
 
 find_package(Boost REQUIRED COMPONENTS program_options)
 
-get_property(LIB64 GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS)
-if ("${LIB64}" STREQUAL "TRUE")
-    set(LIBDIR "lib64")
+if (NOT APPLE)
+    get_property(LIB64 GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS)
+    if ("${LIB64}" STREQUAL "TRUE")
+        set(LIBDIR "lib64")
+    else()
+        set(LIBDIR "lib")
+    endif()
 else()
     set(LIBDIR "lib")
 endif()
 
 function(setup_rpath name)
-  if(APPLE)
-    set_target_properties(${name} PROPERTIES
-                          BUILD_WITH_INSTALL_RPATH ON
-                          INSTALL_RPATH "@loader_path/../${LIBDIR}/trellis"
-                          INSTALL_NAME_DIR "@rpath")
-  elseif(UNIX)
-    set_target_properties(${name} PROPERTIES
-                          BUILD_WITH_INSTALL_RPATH ON
-                          INSTALL_RPATH "\$ORIGIN/../${LIBDIR}/trellis")
-  endif()
+    if(APPLE)
+        set_target_properties(${name} PROPERTIES
+                              BUILD_WITH_INSTALL_RPATH ON
+                              INSTALL_RPATH "@loader_path/../${LIBDIR}/trellis"
+                              INSTALL_NAME_DIR "@rpath")
+    elseif(UNIX)
+        set_target_properties(${name} PROPERTIES
+                              BUILD_WITH_INSTALL_RPATH ON
+                              INSTALL_RPATH "\$ORIGIN/../${LIBDIR}/trellis")
+    endif()
```

Comment:
Just a non-authoritative opinion: I generally like to avoid mixing of indentation cleanup (a no-op change from a functional p.o.v.) with actual functional modifications in the same patch. A separate patch cleaning up indentations (which says so in the commit log) is easy to review by looking for removals and insertions of the same lines with different amounts of leading whitespace. Similarly, a patch that just removes and adds the minimally necessary number of lines to effect a meaningful change is easier to review if one doesn't have to deal with the "false positive" whitespace cleanup changes mixed in... :)
Of course, in this case the entire patch is small enough for that not to be a big deal.